File: app/src/main/java/social/entourage/android/map/entourage/minicards/EntourageMiniCardViewHolder.java
package social.entourage.android.map.entourage.minicards;

import android.view.View;

import social.entourage.android.map.entourage.EntourageViewHolder;

/**
 * View Holder for Entourage mini cards that are shown on heatzone tap<br/>
 * Created by Mihai Ionescu on 05/10/2017.
 * @see EntourageViewHolder
 * @see social.entourage.android.newsfeed.FeedItemViewHolder
 */

public class EntourageMiniCardViewHolder extends EntourageViewHolder {

    public EntourageMiniCardViewHolder(final View itemView) {
        super(itemView);
    }

    @Override
    protected boolean showCategoryIcon() {
        return false;
    }
}

File: app/src/main/java/social/entourage/android/entourage/minicards/EntourageMiniCardViewHolder.kt
package social.entourage.android.entourage.minicards

import android.view.View
import social.entourage.android.entourage.EntourageViewHolder

/**
 * View Holder for Entourage mini cards that are shown on heatzone tap<br></br>
 * Created by Mihai Ionescu on 05/10/2017.
 * @see EntourageViewHolder
 *
 * @see social.entourage.android.newsfeed.FeedItemViewHolder
 */
class EntourageMiniCardViewHolder(itemView: View) : EntourageViewHolder(itemView) {
    override fun showCategoryIcon(): Boolean {
        return true
    }
}